lib/libzfs/libzfs_import.c

```diff
@@ -260,6 +260,91 @@ udev_device_is_ready(struct udev_device *dev)
 }
 
 /*
+ * Wait timeout milliseconds for a newly created device to be available
```

Comment:
I'm good with pedantic, carry on!  The more eyes on this the better.  Easy fix, will do.